homeassistant/components/climate/const.py

```diff
@@ -64,6 +64,22 @@ class HVACMode(StrEnum):
 # Device is reacting to activity (e.g. movement sensors)
 PRESET_ACTIVITY = "activity"
 
+
+class FanState(StrEnum):
```

Comment:
This changes our entity component, for which an architectural discussion is required before changing it.

../Frenck